python/pyspark/sql/context.py

```diff
@@ -63,7 +63,33 @@ def toDF(self, schema=None, sampleRatio=None):
         """
         return sqlContext.createDataFrame(self, schema, sampleRatio)
 
+    def schema(self, datatype):
```

Comment:
How will this compared to `toDF` and `createDataFrame`, should we make them identical?